### ISSUE TITLE
resize icon does not show up on bottom right corner

### ISSUE THREAD

**Issue by bvkavya** [Adobe-CEP/CEP-Resources#240]
![issue](https://user-images.githubusercontent.com/42142724/59998707-bdbefe80-967d-11e9-8f0d-c0fbd489225a.png)

Hi,

As you can see in the attached screenshot, there is a small box appearing on the plugin's bottom right corner. This is happening on windows 10. Can this be replaced with resize symbol? how can we do it? 

Please note, there is no custom code is added. its just a sample plugin I created with index.html as follows
<!doctype html>

<html>
<head>
<script src="./lib/CSInterface-4.0.0.js"></script>

<title>testplugin</title>
</head>

<body>
</body>

</html>

Could you please let me know what causing the issue?

**Comment by Charu16** [comment]
This is the default behaviour of CC panels for all Adobe products.

**Comment by Inventsable** [comment]
You can customize the scrollbar within the panel (including the resizer) with CSS targeting the `::webkit-scrollbar` pseudo-selector, as [in this link](https://css-tricks.com/almanac/properties/s/scrollbar/)

**Comment by bvkavya** [comment]
> You can customize the scrollbar within the panel (including the resizer) with CSS targeting the `::webkit-scrollbar` pseudo-selector, as [in this link](https://css-tricks.com/almanac/properties/s/scrollbar/). As far as I know all CSS is valid including `background-image`, `width` and `height` certainly are

Thanks for your reply.
I tried this. Whatever background image I set, it is hiding behind that small box from the screenshot.


> This is the default behaviour of CC panels for all Adobe products.

Thanks for your reply.
However, it does not happen on Mac. Only windows have this issue. I have a plugin developed on Flex code which does not have this issue. If it is the default behavior could you please share me any link where it is mentioned about this default behavior.

**Comment by Charu16** [comment]
Yes, it will never come above the square box, because whatever CSS you will apply it will apply on the body of the panel and that square box is not a part of HTML, it is a part of the panel.

And yes, it happens only on Windows and I think this is not an issue it is a feel and looks of the panel that differs based on OS. Flex is an old story. You can not compare FLEX with HTML. There are lots of difference in CS version and CC. This can be validated only by creating a panel. You can check CS6 version panels have resize icon and CC version does not. Here is the screenshot of Layers panel from Indesign application.

<img width="416" alt="CS6 version" src="https://user-images.githubusercontent.com/41332091/60155639-4ff30e00-9808-11e9-841f-fe670fb10ca9.png">

<img width="265" alt="CC version" src="https://user-images.githubusercontent.com/41332091/60155645-54b7c200-9808-11e9-99e1-007d5243d191.png">


You can ask Adobe team to change panels look and feel for you to do same as CS6 :)



Thanks

**Comment by acswift** [comment]
Still an issue, and it happens on Mac too. I would say that it's there about 50% of the time.

At least on Mac, it definitely feels like a bug. It's not consistant, and there's no little grippy texture to show the user what it's for.

As I'm writing this, I thought perhaps it's related to a panel being resizable in the manifest.xml:

1. I made the maximum size the same as the minimum size and restarted Illustrator
2. the gray square disappeared
3. I undid the modification in (1), and the gray square is still gone

Not sure what to conclude except that the behavior doesn't seem intentional.

I will try starting with the panel maximized.
